dvc/scm/git.py

```diff
@@ -372,14 +379,18 @@ def remind_to_track(self):
         if not self.files_to_track:
             return
 
-        files = " ".join(shlex.quote(path) for path in self.files_to_track)
+        if self.autostage:
```

Comment:
I think, it's better to create `track_changed_files()` that does this and call from `scm_context`, something like this
```python
auto_stage = repo.config("core", {}).get("autostage", False)
if auto_stage:
  repo.scm.track_changed_files()
else:
  repo.scm.remind_to_track()
```

Reply:
This way, you don't need to drill down the `autostage` too.